test/parallel/test-repl-tab-complete.js

```diff
@@ -24,6 +24,7 @@
 const common = require('../common');
 const assert = require('assert');
 const fixtures = require('../common/fixtures');
+const hasInspector = process.config.variables.v8_enable_inspector === 1;
```

Comment:
This doesn't seem to be used?

Reply:
I'm going to use it when I figure out how to make the test work